src/ownership/lifetimes-function-calls.md

```diff
@@ -1,6 +1,13 @@
 # Lifetimes in Function Calls
 
-In addition to borrowing its arguments, a function can return a borrowed value:
+_Example problem_: 
+  * Two references to two values are borrowed by a function and the function returns
```

Comment:
The text is good, but I feel it would be too much for the slide. It's the kind of introductory explanation that the instructor should give, but the students should not stare at the text while the instructor reads it.

In other words: can we move this to the notes, please?

Reply:
Done